float32 voltage
int8 percentage
float32 current
int16 remaining_time
float32 integrated_current
